### ISSUE TITLE
Chatmon blocking "//send @others setkey numpad8 down"

### ISSUE THREAD

**Issue by Schliematt** [Windower/Issues#938]
I dual box and since the creator's update shenanigans Ive been having problems with exec and setkeys when sent to the background character for a while now. But I was out of town til this week and just attempted to see if it was any addons/plugins blocking it from working since everyone else says setkeys and execs work for them.

I unloaded every addon and plugin and the setkeys worked, so obviously something was blocking it. So i loaded all the addons and setkey still worked. So went thru the 10 or so plugins i still use and came up with Chatmon was the culprit.

While loaded on the front (focused) window it seems to work, but having Chatmon loaded on the background window prevents the setkey from activating. Without further commands, If i switch over (alt+tab) to the background window it will take effect, so its like it receives the setkey, but doesnt let the setkey actually work (at least not until that window comes into focus). You can send the command to Window B (in the background) from Window A, see nothing happen, switch over to Window B, it starts running, switch back to Window A, see the Window B's character stop on your screen, switch back to Window B's character and they start running again. Setkey only works while in focus with Chatmon loaded.

I tested with these commands in a flat area where my mule could run freely (port sandoria by AH):
run forward
"//send @others setkey numpad8 down"
"//send @others setkey numpad8 up"
run backward
"//send @others setkey numpad2 down"
"//send @others setkey numpad2 up"

**Comment by joshk6656** [comment]
Unload chatmon and see if it's fixed. Someone posted that's causing the
issue, but we have not confirmed or looked at the code yet.

**Comment by Schliematt** [comment]
Pretty much at the beginning of the month I was told that I was the only one having this issue now. So didnt look like anyone would be able to correct it if they couldnt replicate it. I originally added to the first post cause it was setkey related and didnt know the cause. When I posted the other day I was kinda hoping for a "yes we can replicate this problem now" or some kind of reply. But no big deal, I know people are busy and whatnot.

But since I found this was actually chatmon related I thought it might deserve its own issue post. And maybe some kind of reply, which it did so mission accomplished. I prolly would have figured out it was chatmon related issue if i wasnt out of town all month. My sub ends tonight and it might be a bit before i can pay for it so might not be able to test myself if its fixed if your guys release a fix for it.

Good to know its in the works at some point and thank you guys for your time.
